### DISCUSSION TITLE
First Day of Week to current day?

### DISCUSSION THREAD

**Discussion by Qwarkk6** [MMRIZE/MMM-CalendarExt3#11] [Q&A]
I was hoping to use this module to show what is on my calendar for the next 7 days without ever seeing past days/events, but that doesn't seem possible at the moment as you lock in the first day of the week. Is there a way to do what I'm trying to do?

**Comment by eouia** [comment]
`MMM-CalendarExt2` module(the ancestor of this module) has a view for that purpose - `Daily` view. But very few people had an interest in that view. So I remove that view from this module. Sorry.
